src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductTaxon.xml

```diff
@@ -16,44 +16,23 @@
             xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd">
 
     <class name="Sylius\Component\Core\Model\ProductTaxon">
-        <attribute name="id">
-            <group>admin:product_taxon:index</group>
-            <group>sylius:admin:product_taxon:index</group>
-            <group>admin:product_taxon:show</group>
-            <group>sylius:admin:product_taxon:show</group>
-            <group>shop:product_taxon:show</group>
-            <group>sylius:shop:product_taxon:show</group>
-        </attribute>
```

Comment:
Why are you removing this id? It is the only identifier of this resource, I would be for leaving it as it was